srcalc/src/main/java/gov/va/med/srcalc/domain/model/DerivedTerm.java

```diff
@@ -13,9 +17,14 @@
  * <p>Per Effective Java Item 17, this class is marked final because it was not
  * designed for inheritance.</p>
  */
+@Embeddable
 public final class DerivedTerm extends ModelTerm
 {
-    private final Rule fRule;
+    private Rule fRule;
+    
+    DerivedTerm()
```

Comment:
Please add a Javadoc indicating that this constructor is intended for reflection-based construction only.